src/cswinrt/code_writers.h

```diff
@@ -5289,20 +5289,18 @@ else
         for (auto&& evt : iface.EventList())
         {
                     w.write(R"(%
-% static unsafe (Action<%>, Action<%>) Get_%(% %, object _thisObj)
+% static unsafe global::WinRT.Interop.EventSource<%> Get_%(% %, object _thisObj)
```

Comment:
Wouldn't dropping the old one be a breaking change?

Reply:
So, my thinking is that this is not a breaking change because as of the current CsWinRT (as in, pre-AOT, ie. 2.0.6), these methods are in the various generated `<TYPE_NAME>Methods` classes, which are generated as internal (looking at eg. Win2D with ILSpy). So we should be able to change their signatures without affecting public consumers, since those would just go through the public events on the projected types instead, not the interop methods. What do you think?

Reply:
I agree with if this is within a class that is internal, then just we can drop the old definition and just have the new one.  But you will see there are a couple scenarios that don't fall into this case.  i.e. ones which are public interfaces and have events on them such as `ABI.Windows.UI.Core.ICoreAcceleratorKeysMethods`.  For those, I think we will need to have both the old method and the new one to avoid this being a breaking change.  For new generated projections, we will use the new one and old ones are not broken as the old one is still there.  Thoughts?